data_pipeline/pipeline.py

```diff
@@ -58,11 +58,13 @@ def get_s3_object(bucket, key):
 
 @autokitteh.activity
 def insert_records(db_dsn, records):
-    with closing(sqlite3.connect(db_dsn)) as conn, conn:
+    with closing(sqlite3.connect(db_dsn)) as conn:
         cur = conn.executemany(INSERT_SQL, records)
+        conn.commit()
```

Comment:
This should be the same as the previous code, but if it fixes the issue - go for it.